client/src/App.jsx

```diff
@@ -14,16 +17,16 @@ class App extends Component {
       <div className="App">
         <Router>
           <Switch>
-            <Route path="/login" component={Login} exact />
+            {/* <Route path="/login" component={Login} exact /> */}
```

Comment:
I think we need to work on API route tomorrow to avoid all conflicts and comment lines in this file.